pyproject.toml

```diff
@@ -34,7 +34,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.7"
 aiohttp = "^3.6"
-pydantic = "^1.5"
+pydantic = "^1.5.1"
```

Comment:
Still waiting for 1.5.2 or 1.6... hrr
Not all bugfixes is released with 1.5.1

Reply:
Got it. Gotta rollback our pydantic-related related changes.
Tests are passing with 1.5.1 tho.

Reply:
resolved.